### ISSUE TITLE
Nothing is generated

### ISSUE THREAD

**Issue by Zelmoghazy** [fcying/compiledb-go#3]
```
❯ compiledb make

mkdir build
avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra main.c -o build/main.o
avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra src/gpio.c -o build/gpio.o
avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra src/i2c.c -o build/i2c.o
avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra src/spi.c -o build/spi.o
avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra src/uart.c -o build/uart.o
avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra HAL/lcd.c -o build/lcd.o
```
no compile_commands.json is generated , using version:v1.3.6

**Comment by fcying** [comment]
Could you provide the detailed log from `compiledb -nv make`?  I tested the build.log works correctly.
```
  {
    "directory": "/opt/compiledb_test/src",
    "arguments": [
      "avr-gcc",
      "-c",
      "-mmcu=atmega328p",
      "-gdwarf-2",
      "-I.",
      "-I./include/",
      "-I./HAL/",
      "-O0",
      "-DF_CPU=8000000UL",
      "-DBAUD=9600",
      "-Wall",
      "-pedantic",
      "-Wextra",
      "main.c",
      "-o",
      "build/main.o"
    ],
    "file": "main.c"
  },
```

**Comment by Zelmoghazy** [comment]
> compiledb -nv make

time="2024-10-07T01:14:44+03:00" level=debug msg="New command:avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra HAL/lcd.c -o build/lcd.o"
time="2024-10-07T01:14:44+03:00" level=warning msg="file /c/Users/{path}/HAL/lcd.c not exist"
time="2024-10-07T01:14:44+03:00" level=debug msg="New command:avr-gcc -c -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra HAL/max7221.c -o build/max7221.o"
time="2024-10-07T01:14:44+03:00" level=warning msg="file /c/Users/{path}/HAL/max7221.c not exist"
time="2024-10-07T01:14:44+03:00" level=debug msg="New command:avr-gcc build/main.o build/gpio.o build/i2c.o build/spi.o build/uart.o build/lcd.o build/max7221.o -o build/atmega32.elf -mmcu=atmega328p -gdwarf-2 -I. -I./include/ -I./HAL/ -O0 -DF_CPU=8000000UL -DBAUD=9600 -Wall -pedantic -Wextra -Wl,--gc-sections -Wl,--relax"
time="2024-10-07T01:14:44+03:00" level=debug msg="New command:avr-objcopy -O ihex -R .eeprom  build/atmega32.elf build/atmega32.hex"
time="2024-10-07T01:14:44+03:00" level=debug msg="New command:echo \"=========================================\""
time="2024-10-07T01:14:44+03:00" level=debug msg="New command:echo \"              BUILD SUCCESS              \""
time="2024-10-07T01:14:44+03:00" level=debug msg="New command:echo \"=========================================\""
time="2024-10-07T01:14:44+03:00" level=debug msg="New command:make: Leaving directory '/c/Users/{path}'"
time="2024-10-07T01:14:44+03:00" level=info msg="change workingDir: C:/Users/{path}"

**Comment by fcying** [comment]
It seems there is an issue with path recognition. `Entering directory '/c/Users/{path}/Lab_6_ATMEGA32_I2C_Driver'`, this type `/c/xxx` of path is not being recognized, so the files cannot be found. I'll look into how to fix it. Thanks for report~

**Comment by fcying** [comment]
@Zelmoghazy fix in 1.3.7, you can try a test

**Comment by Zelmoghazy** [comment]
> @Zelmoghazy fix in 1.3.7, you can try a test

Yes, working perfectly now , thank you
